### ISSUE TITLE
Doesn't work on windows

### ISSUE THREAD

**Issue by TanninOne** [orta/vscode-jest#10]
Reports this error message:
`Exception raised: Process failed: spawn node_modules/.bin/jest ENOENT`

On windows you'd have to spawn node_modules/.bin/jest.cmd

**Comment by orta** [comment]
ah, cool 👍

**Comment by orta** [comment]
I've shipped a new build which adds the ability to customise the Jest path, can you change `jest.pathToJest` to `node_modules/.bin/jest.cmd` and let me know if that fixes it?

**Comment by onemen** [comment]
Now i get this error:
```
[2J[H
No tests found related to files changed since last commit.
Press `a` to run all tests, or run Jest with `--watchAll`.

Error: ENOENT: no such file or directory, open 'c:\tmp\vscode-jest.json'
    at Object.fs.openSync (fs.js:557:18)
    at Object.fs.writeFileSync (fs.js:1222:33)
    at source.getTestPaths.then.then.then.runResults (c:\Users\onemen\yarn\node_modules\jest-cli\build\jest.js:212:14)
```

1. I use Hg to commit changes, not Git, how sould i configure the extension?.
2. I don't have 'c:\tmp' and vscode-jest.json, where should i have vscode-jest.json in windows, and what should be in it?

**Comment by orta** [comment]
Cool, well that's part of the way then 🌮 

1. This is just how Jest works, it watches for changes in hg/git and only runs those tests - if this is new for you, you might want to experiment with just using Jest for a bit to understand the principals 👍 
2. I'd need a path for a temp directory on windows, wonder if the node API has something for this

**Comment by onemen** [comment]
const os = require('os');
os.tmpdir(),

**Comment by orta** [comment]
Thanks, looking at it right now

**Comment by orta** [comment]
Shipped a PR and a _maybe_ working build, can you two take a look?

**Comment by TanninOne** [comment]
Now it appends ".bat" to the jest.pathToJest. The file is called jest.cmd,  not jest.bat though.

To test further I renamed the script and now jest gets run properly and errors are displayed inline so it seems to work fine now 👍

One thing I noticed: The option jsonOutputFile is extremely new so it might be worth mentioning in the docs that this requires a current version of jest (15.x was too old, 17.x works).
